packages/rancher-sachet/charts/values.yaml

```diff
@@ -0,0 +1,105 @@
+# Default values for sachet.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  cattle:
+    systemDefaultRegistry: ""
+  preInstallHook:
+    securityContext:
+      runAsNonRoot: true
+      runAsUser: 1000
+    image:
+      repository: rancher/rancher-agent
+      tag: v2.5.7
+      pullPolicy: IfNotPresent
+  namespaceOverride: ""
+
+nameOverride: "sachet"
```

Comment:
`.Values.nameOverride` and `.Values.fullnameOverride` are from the default Helm template created by the `helm template` command, and they are used in the named templates for determining the fullname/name of all resources. 

@aiyengar2 Do I answer your question?

Reply:
The name is provided as part of the helm command itself, e.g.

```
helm install -n <namespace> <.Release.Name>
```

With this approach, we would be ignoring the .Release.Name always, which means I cannot deploy rancher-alerting-drivers twice into the same namespace right?

Doubt that will be required, but our versioning scheme shouldn't prevent something that it can avoid preventing (e.g. by naming resources as `{{ .Release.Name }}-sachet` instead, if we don't want `{{ .Release.Name }}` since that can conflict with other installed drivers).

Reply:
We can deploy `rancher-alerting-drivers` twice into the same namespace by giving them a different release name because the `nameOverride` is used as the value of the label `app.kubernetes.io/name` on pods and pods selectors, which does not affect the name of the deployment/pod/service or any other resources of the app. 

Now I set the default to be an empty string (`nameOverride: ""`), so the value of `.Chart.Name` is used for the above label. 

`nameOverride` and `fullnameOverride` are fields from Helm's default chart template (and also in the upstream prom2teams chart), I believe Helm needs them for some more complex use cases than ours.